plugins/pastefromword/plugin.js

```diff
@@ -7,7 +7,7 @@
 	/* global confirm */
 
 	CKEDITOR.plugins.add( 'pastefromword', {
-		requires: 'clipboard',
+		requires: 'clipboard,ajax',
```

Comment:
Isn't it any other way to not require `ajax` for PfW? It looks a little bit strange. I wonder if `FileReader` would be an alternative here.

Reply:
I've considered using `FileTools` here. Unfortunately `FileTools` contains logic responsible for uploading files to the server, there is no methods helping downloading files from given URL, where `Ajax` has it and only requires to add new file type to support. That's it was chosen here. I could extend `FileTools`, with new functions, but this seems like new separate task, or hide completely XHR inside PFW and do not rely on `Ajax` plugin.
Using `Ajax` seems to be most convenient and time efficient for me.

@mlewand WDYT? Should we require Ajax in PFW or use some solution?

Reply:
@Comandeer what are the downsides of using `ajax` plugin for this case? Using FileReader just for sake of using it results with more code to worry about.

Reply:
@mlewand it wasn't exactly a downside, I just felt that current use of ajax plugin is inappropriate, but I haven't spend much time on it as whole fix wasn't working for me. You're comment about using plugin in core explains my anxiety ;)